module_4/config/dbConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "112233",
+    "database": "postgres",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "port": "5432"
```

Comment:
this is something that must be taken form `process.env`

Reply:
updated, thanks